src/sdl2_net-1-pkg-config.patch

```diff
@@ -0,0 +1,80 @@
+From e2782ccaa642202e50aebefe4107e4313f6bbb53 Mon Sep 17 00:00:00 2001
+From: Cameron Cawley <ccawley2011@gmail.com>
+Date: Sat, 17 Jul 2021 15:56:53 +0100
+Subject: [PATCH] Include required libraries in pkg-config output
+
+---
```

Comment:
Please include "Taken from.." here.
Also, the stats aren't necessary. The script tools/patch-tool-mxe can be used to "normalize" patch format.

Reply:
Done.